src/components/ArrivalDeparture.svelte

```diff
@@ -38,7 +38,7 @@
 			};
 		} else if (scheduledDiff <= 0) {
 			return {
-				status: 'on time',
+				status: `${$t('status.on_time')}`,
```

Comment:
I'll still merge this, but is there any reason this isn't expressed as:

```
status: $t('status.on_time'),
```

isn't the outer template string redundant?

Reply:
you're right I will update it in the upcoming PR.